docs/source/glossary.rst

```diff
@@ -0,0 +1,164 @@
+    different types of consensus on the same blockchain.
+    For more information, see :ref:`pluggable-consensus-label`.
+
+  Core
+    See :term:`Sawtooth core`.
+
+  Distributed ledger
+    See :term:`Blockchain`.
+
+  Genesis block
+    First block on the blockchain. The genesis block initializes the
+    Sawtooth network.
+
+  Global state
+    Database that stores a local (validator-specific) record of transactions for
+    the blockchain. Sawtooth represents state in a single instance of a
+    Merkle-Radix tree on each validator node.  For more information, see
+    :doc:`architecture/global_state`.
+
+  Identity
+    Sample transaction family that handles on-chain permissions (settings
+    stored on the blockchain) for transactor and validator keys. This
+    transaction family demonstrates how to streamline managing identities
+    for lists of public keys. For more information, see
+    :doc:`transaction_family_specifications/identity_transaction_family`.
+
+  IntegerKey
+    Sample transaction family with only three operations (set, increment, and
+    decrement) that can be used to test deployed ledgers. For more information,
+    see :doc:`transaction_family_specifications/integerkey_transaction_family`.
+
+  Journal
+    Sawtooth core component that is responsible for maintaining and extending
+    the blockchain for the validator. For more information, see
+    :doc:`architecture/journal`.
+
+  Merkle-Radix tree
+    Addressable data structure that stores state data. A Merkle-Radix tree
+    combines the benefits of a Merkle tree (also called a "hash tree"), which
+    stores successive node hashes from leaf-to-root upon any changes to the
+    tree, and a Radix tree, which has addresses that uniquely identify the
+    paths to leaf nodes where information is stored.  For more information, see
+    :ref:`merkle-radix-overview-label`.
+
+  Node
+    Participant in Sawtooth network. Each node runs a single validator, a
+    REST API, and one or more transaction processors.
+
+  Off-chain setting
+    Setting or value that is stored locally, rather than on the blockchain.
+
+  On-chain setting
+    Setting or value that is stored on the blockchain (also referred to as
+    "in state") so that all participants on the network can access that
+    information.
+
+  Permissioned network
+    Private network of Sawtooth nodes. The blockchain stores the settings that
+    specify permissions, such as roles and identities, so that all participants
+    in the network can access this information.
+
+  PoET
+    Proof of Elapsed Time, a Nakamoto-style consensus algorithm that is designed
+    to support large networks. PoET relies on a trusted execution environment
+    in |Intel (R)| CPUs called Software Guard Extensions (SGX).  For more
+    information, see :doc:`architecture/poet`.
+
+  REST API
+    Component that adapts communication between the client application and
```

Comment:
REST API is a general term with meaning outside of Sawtooth, and refers to a specific sort of HTTP/JSON interface. I'm not sure describing it as a general sort of adaptor between clients and validators is accurate. You could write client adaptors that were not REST APIs, or you could have a REST API that does not talk to a validator. I would suggest rewriting this paragraph something like this:

"In relation to Sawtooth, refers to a core component which adapts communication with a validator to HTTP/JSON standards. This can simplify application development, and is used by clients such as the Sawtooth CLI. Developers may use this REST API or develop their own more specific APIs for client/validator communication."